app/components/access-level-pill.hbs

```diff
@@ -27,30 +27,34 @@
         </div>
         <button
```

Comment:
Waarom wordt hier geen <AukButton> gebruikt?

Reply:
Hier ben ik niet 100% mee vrees ik.

Reply:
Sorry, de verkeerde lijn is geselecteerd en github is ook niet mee met de tag...

Ik bedoelde: waarom wordt hier geen Webcomponents::Button gebruikt?

Reply:
Op lijn 39 staat een rechtstreekse button, we hebben hier een component voor, waarom niet overal het component gebruiken?

```
<button type="button"
          class="auk-button auk-button--borderless auk-button--icon"
          data-test-access-level-save
          {{on "click" (perform this.confirmChangeAccessLevel @accessLevel)}}
        >...
```